homeassistant/components/tplink/manifest.json

```diff
@@ -24,36 +24,68 @@
       "macaddress": "98DAC4*"
     },
     {
-      "hostname": "lb*",
+      "hostname": "hs*",
+      "macaddress": "B09575*"
+    },
+    {
+      "hostname": "kl*",
       "macaddress": "1C3BF3*"
     },
     {
-      "hostname": "lb*",
+      "hostname": "kl*",
       "macaddress": "50C7BF*"
     },
     {
-      "hostname": "lb*",
```

Comment:
Would it make sense to remove the hostname filtering altogether? Looking at this specific patch, it may very well be that there are kl and lb bulbs using the same prefix.

Reply:
I thought about that but I don't know for sure that tplink won't/hasn't used the OUI for other non-kasa devices.